parsec/api/protocol/invite.py

```diff
@@ -48,6 +48,7 @@ class InvitationToken(UUID4):
 class InvitationType(Enum):
     USER = "USER"
     DEVICE = "DEVICE"
+    PKI = "PKI"
```

Comment:
Sounds good to me! As I said elsewhere it's still a bit like an invitation to me, in the sense that the user is invited to perform the pki enrollment procedure, but the `BackendPkiEnrollmentAddr` naming sounds quite explicit about that.

The original reason for this choice was to avoid code duplication (as those `BackendSomeActionAddr` are not trivial to implement), but keeping things separated is probably for the better.